cmd/oci-flexvolume-driver/main.go

```diff
@@ -16,11 +16,12 @@ package main
 
 import (
 	"fmt"
-	"log"
 	"os"
 
 	"github.com/oracle/oci-cloud-controller-manager/pkg/flexvolume"
 	"github.com/oracle/oci-cloud-controller-manager/pkg/flexvolume/block"
+	"github.com/oracle/oci-cloud-controller-manager/pkg/logging"
+	"go.uber.org/zap"
```

Comment:
nit: import grouping (internal packages vs 3rd party)

Reply:
I feel like VS code occasionally does this automatically... at least now I know.

Reply:
It should sort them automatically but you need to put in the blank line to create the groups 😄